packages/hardhat-core/src/internal/core/jsonrpc/types/input/debugTraceTransaction.ts

```diff
@@ -5,6 +5,8 @@ import { optionalOrNullable } from "../../../../util/io-ts";
 export const rpcDebugTracingConfig = optionalOrNullable(
   t.type(
     {
+      tracer: optionalOrNullable(t.string),
+      callTracer: optionalOrNullable(t.boolean),
```

Comment:
I don't hink there's a `callTracer` option?

You can see the supported options [here](https://geth.ethereum.org/docs/interacting-with-geth/rpc/ns-debug#debugtracetransaction).